lib/kakine/config.rb

```diff
@@ -46,7 +46,7 @@ def self.validate_config
     def self.setup_yao
       Yao.configure do
         auth_url    @@config['auth_url']
-        tenant_name @@config['tenant_name']
```

Comment:
Now that `@@config['tenant_name']` is not used, other routines related to `tenant_name` in this file can be deleted.